src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -218,6 +218,181 @@ void uct_rc_mlx5_iface_common_tag_cleanup(uct_rc_mlx5_iface_common_t *iface,
 #endif
 }
 
+
+#if HAVE_IBV_EXP_DM
+const uint32_t uct_rc_mlx5_dm_key = 1245; /* magic number */
+
+typedef struct {
+    uct_mlx5_dm_data_t *dm;
+    int                count;
```

Comment:
why need this field? AFAIU mpool already maintains the count..

Reply:
it is used to calculate offset from begin of DM buffer for every desc